testproject/Assets/Scripts/GrabbableBall.cs

```diff
@@ -100,7 +100,6 @@ private void ReleaseServerRpc()
         if (m_IsGrabbed.Value)
         {
             NetworkObject.RemoveOwnership();
-
```

Comment:
If this is the only change to that file, it's probably better to leave it off, no?